services/sky/document_view.cc

```diff
@@ -302,15 +332,8 @@ void DocumentView::StartDebuggerInspectorBackend() {
 }
 
 void DocumentView::InitServiceRegistry() {
-  mojo::ConnectToService(imported_services_.get(), &service_registry_);
-  mojo::Array<mojo::String> interface_names(1);
-  interface_names[0] = mojo::ViewManagerClient::Name_;
-  mojo::ServiceProviderImpl* sp_impl(new mojo::ServiceProviderImpl());
-  sp_impl->AddService(&view_manager_client_factory_);
-  mojo::ServiceProviderPtr sp;
-  service_registry_service_provider_binding_.reset(
-      new mojo::StrongBinding<mojo::ServiceProvider>(sp_impl, &sp));
-  service_registry_->AddServices(interface_names.Pass(), sp.Pass());
+  if (imported_services_)
+    mojo::ConnectToService(imported_services_.get(), &service_registry_);
```

Comment:
It's not unreachable.  This code may still be used by Modular.  Note that the imported services are coming from the connecting app via the shell, not the embedder, so it's still relevant whether the connecting app passed a service registry.

I'd like to revisit this in a later patch.